blaze/config/client.py

```diff
@@ -130,7 +130,7 @@ def get_default_client_environment():
     )
 
 
-def get_client_environment_from_parameters(bandwidth: int, latency: int, cpu_slowdown: int):
+def get_client_environment_from_parameters(bandwidth: int, latency: int, cpu_slowdown: int, loss: int):
```

Comment:
This change is going to break other calls of this function (see the broken tests/lint)

Reply:
One way to solve this is to provide a default value